waffle-hardhat/src/inject-call-history.ts

```diff
@@ -0,0 +1,61 @@
+import {waffle} from 'hardhat';
+import type {RecordedCall} from 'ethereum-waffle';
+import {utils} from 'ethers';
+
+class CallHistory {
```

Comment:
How about defining an `ICallHistory` interface in waffle-provider, importing here (as type from dev dep), so that the interface is kept in sync?